js/sfx.js

```diff
@@ -0,0 +1,11 @@
+import {ZZFX, zzfx} from 'zzfx'
+
+// you can use https://killedbyapixel.github.io/ZzFX/ to generate sfx
+
+export const playExplosionSFX = ()=> {
+    if (Math.random() >=.5){
+        zzfx(...[1.01,,690,.02,.3,.55,4,3.03,.8,,,,,1.8,,.2,,.5,.12,.11])
```

Comment:
it will be good to move all our sounds to this type, so we can remove the howler dependency and not to have two dependencies for sounds